wagtail_content_import/parsers/tests.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+
+from .google import GoogleDocumentParser
+
+import json
```

Comment:
We should put standard library imports at the top

Reply:
Thanks, I've fixed.